README.md

```diff
@@ -115,6 +115,10 @@ Follow the steps here and configure the presto-yarn configuration files to match
     "site.global.plugin": "{'ml': ['presto-ml-${presto.version}.jar']}",
 ```
 
+* ``site.global.app_name`` (optional) (default - ``presto-server-0.130``) This value should be the name of the tar.gz file contained within the zip file produced by presto-yarn (in package/files/ within the zip). If you use a custom presto server distribution or anything other than the default presto-yarn package settings, please be sure to modify this.
+
+* ``application.def`` For Slider users, when the command the install the presto package is run, the logs will explicitly tell the user which value to use for this parameter.
```

Comment:
typo: "command to install".

Also, we should mention here that changing this is required only if you are using a custom built package. If using the package built from the repository, this is not required.